jetty-jmh/src/main/java/org/eclipse/jetty/http/jmh/MultiPartBenchmark.java

```diff
@@ -188,23 +188,29 @@ public long testLargeGenerated() throws Exception
                 case "HTTP":
                 {
                     MultiPartFormInputStream parser = new MultiPartFormInputStream(in, _contentType, config, outputDir.toFile());
```

Comment:
Is it possible to setup a null/no-op destination for MultiPartFormInputStream?  something that doesn't use a real filesystem?
We want to test the performance of that algorithm, not our filesystem.

Reply:
will that work on windows if using /dev/null?